packages/fiori/src/Bar.js

```diff
@@ -133,6 +143,49 @@ class Bar extends UI5Element {
 			"label": this.design,
 		};
 	}
+
+	constructor() {
+		super();
+
+		this._handleResizeBound = this.handleResize.bind(this);
+	}
+
+	handleResize() {
+		const bar = this.getDomRef();
+		let changeFlex = false;
+		const barWidth = bar.offsetWidth;
+
+		changeFlex = Array.from(bar.children).some(element => {
```

Comment:
I think you can further simplify it by using this:

this._shrinked = Array.from(bar.children).some(element => {
    return barWidth / 3 < element.offsetWidth;
});


instead of:

changeFlex = Array.from(bar.children).some(element => {
        if (barWidth / 3 < element.offsetWidth) {
				return true;
			}

			return false;
		});

		if (changeFlex) {
			this._shrinked = true;
		} else {
			this._shrinked = false;
		}
	}